src/main/java/md/maib/entity/Customer.java

```diff
@@ -0,0 +1,101 @@
+package md.maib.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class Customer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    private Long id;
+
+    @Column(name = "cvv", nullable = false)
+    private String cvv;
+
+    @Column(name = "username", nullable = false, unique = true)
+    private String name;
+
+    @Column(name = "pan", nullable = false, unique = true)
+    private String email;
+
+
+
+    public Customer() {
+    }
+
+    // Builder pattern
```

Comment:
do not use comments in the code, they often indicate that the code needs to be explained, which means the code is too overengineered - this is not good

Reply:
you can write libraries for public use, then comments in the code are your best friends